lms/djangoapps/certificates/signals.py

```diff
@@ -149,6 +149,15 @@ def fire_ungenerated_certificate_task(user, course_key, expected_verification_st
     message = u'Entered into Ungenerated Certificate task for {user} : {course}'
     log.info(message.format(user=user.id, course=course_key))
 
+    if is_using_certificate_allowlist_and_is_on_allowlist(user, course_key):
+        log.info('{course} is using allowlist certificates, and the user {user} is on its allowlist. Attempt will be '
+                 'made to generate an allowlist certificate.'.format(course=course_key, user=user.id))
+        generate_allowlist_certificate_task(user, course_key)
+        return
```

Comment:
Do we need to pass back a status here?

Without it we may miss out on a few logging statements that depend on the return status of `fire_generated_certificate_task` [here](https://github.com/edx/edx-platform/blob/44d97d72f1cf7750815fffffa23c25d47070c9a2/lms/djangoapps/certificates/signals.py#L115) and [here](https://github.com/edx/edx-platform/blob/44d97d72f1cf7750815fffffa23c25d47070c9a2/lms/djangoapps/certificates/signals.py#L74)?

Reply:
Good point. I added that.